addons/web/static/src/views/form/form_controller.scss

```diff
@@ -471,6 +471,20 @@
                 width: auto;
             }
         }
+
+        .o_field_widget {
+            &.o_field_many2one_avatar {
+                display: block;
+            }
+
+            &.o_field_many2many_tags_avatar {
+                display: inline;
+
+                .o_tag_badge_text {
+                    line-height: 1;
+                }
```

Comment:
Not sure what's the relation with the current fix...

Reply:
Actually, for testing purpose I have added a field below many2many avatar field using studio to test the flickering and by setting line-height to 1 i am not getting any flicker. 
video for reference - [link](https://awevideo.s3.amazonaws.com/video-34542597-7a357afcd57a21badf38713570bb2649.webm?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJSCJQ2NM3XLFPVKA%2F20241211%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20241211T130305Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-Amz-Signature=66045a7425e2bc2f58851b2a9a7e14bb007dd3079323307a4a09f84988ee9891)

Reply:
OK. I see your point. Please note, that it also happens in an editable list view, and it looks like the `padding` has also (or even more?) an influence on this issue.

Reply:
Hello @pparidans 
Instead of setting the line-height to 1, we can also decrease the padding of o_tag_badge_text to get the same results.

> Please note, that it also happens in an editable list view,

I'm not getting this part of your message. Can you please explain me?
Thanks!!